packages/valory/skills/decision_making_abci/models.py

```diff
@@ -132,7 +132,7 @@ def get_next_user_id(self):
         """Calculate next user id"""
         if not self.data["users"]:
             return "0"
-        user_ids = list(sorted(list(self.data["users"].keys())))
```

Comment:
This was a bug: before sorting, we need to convert to int or the sorting will be incorrect and therefore we will calculate a wrong ID for new users.